### ISSUE TITLE
display current basal rate from andoidaps

### ISSUE THREAD

**Issue by dexterbooty** [nightscout/nightguard#149]
One thing which I think will be a great addition is to show the current basal rate from androidaps including the percentage if applicable e.g TB: 0.25 (90%)
 Similar to the basal pill on the nightscout website but with the addition of what the percentage of tbr applied. 

![image](https://user-images.githubusercontent.com/10778986/113051171-958ef880-919d-11eb-9c80-fd71cd47561f.jpeg)

**Comment by dhermanns** [comment]
This should already been displayed, right?

![image](https://user-images.githubusercontent.com/3207317/116276580-f0a81100-a784-11eb-9289-86960ce58fde.png)

**Comment by dexterbooty** [comment]
Doesn't seem to show for me but does show in the nightscout site. I am using AndroidAps and Roche combo pump not sure if that is the cause. I've sent you over nightscout it's as usually I have a temp basal running as using smb.

**Comment by dhermanns** [comment]
Sorry - missed that this case is still open. Just had a look. Your extended device status is missing the BaseBasalRate:
/api/v1/devicestatus.json

That is used from nightguard to calculate the basal-rate percentage.

**Comment by dhermanns** [comment]
Did you set DEVICESTATUS_ADVANCED="true"

(https://nightscout.github.io/nightscout/setup_variables/)

**Comment by dexterbooty** [comment]
Hi no problems appreciate it’s a free app and this functionality is a nice to have. 
double checked and the device status-advanced is set to true. I can also see the current basal rate (including temp target changes) being shown in the nightscout basal pill.

**Comment by ideaweb** [comment]
Same problem... DEVICESTATUS_ADVANCED="true" is set (14.2.5) but with Combo pump BaseBasalRate does not exist and in Nightguard just "TB --" is shown. 

```
[
  {
    ...
    "extended": {
        "Version": "3.0-rc3-617cdd681-2022.01.28-17:25",
        "ActiveProfile": "Standard",
        "TempBasalAbsoluteRate": 0.12,
        "TempBasalPercent": 50,
        "TempBasalRemaining": 30
      },
      ...
```

A workaround could be to determine basal from /api/v1/profile/?

**Comment by tast3rmonial** [comment]
Dear ,

We are excited to give you an update on Tastermonial.

Today we have initiated a way to evaluate food claims and for customers to get a prescription for continuous glucose monitor (CGM) sensors and experiment with different foods using our app. 

We are actively working to improve your user experiences in:

👉 Ordering a CGM

We realized the uncertainty of CGM prices in major pharmacies around the US. Therefore, our team is now working on a solution to order CGM online and ship directly to you at a competitive price.

👉 Learning from your CGM food experiment data

[See video of current app's abiity](https://youtu.be/xtaoSxdqq6U)

Here's a peek into upcoming Tastermonial app's upgrade in the next few weeks:

- the ability to run food pairing experiments wtih CGM
- automatic syncing of CGM data to view and share the blood sugar experiments
- You'll be able to use our app to scan and evaluate more foods in your pantry (we're expanding our food database)
 We'd be happy to gather your feedback that could help us in prioritizing the next launches. Click the button below to let us know your needs.

[Customer ***@***.***&firstname=&lastname=)

 

Thanks for reading!

Bude & Tastermonial Team

[Facebook](https://www.facebook.com/tastermonial)
[Instagram](https://www.instagram.com/tastermonial)

No longer want to receive these emails? [Unsubscribe](https://manage.kmail-lists.com/subscriptions/unsubscribe?a=TvyBXH&c=01FTTQGV69XDZXV961Z4NGEACZ&k=33707fbd51467ac523e6126abf1b150b&m=WUxwcT&r=RaxVCBj).
TASTERMONIAL

**Comment by tast3rmonial** [comment]
Dear ,

We are excited to give you an update on Tastermonial.

Today we have initiated a way to evaluate food claims and for customers to get a prescription for continuous glucose monitor (CGM) sensors and experiment with different foods using our app. 

We are actively working to improve your user experiences in:

👉 Ordering a CGM

We realized the uncertainty of CGM prices in major pharmacies around the US. Therefore, our team is now working on a solution to order CGM online and ship directly to you at a competitive price.

👉 Learning from your CGM food experiment data

[See video of current app's abiity](https://youtu.be/xtaoSxdqq6U)

Here's a peek into upcoming Tastermonial app's upgrade in the next few weeks:

- the ability to run food pairing experiments wtih CGM
- automatic syncing of CGM data to view and share the blood sugar experiments
- You'll be able to use our app to scan and evaluate more foods in your pantry (we're expanding our food database)
 We'd be happy to gather your feedback that could help us in prioritizing the next launches. Click the button below to let us know your needs.

[Customer ***@***.***&firstname=&lastname=)

 

Thanks for reading!

Bude & Tastermonial Team

[Facebook](https://www.facebook.com/tastermonial)
[Instagram](https://www.instagram.com/tastermonial)

No longer want to receive these emails? [Unsubscribe](https://manage.kmail-lists.com/subscriptions/unsubscribe?a=TvyBXH&c=01FTTNRAWGB84SXT9WGDAWD06E&k=212e2b001d52af8e7254af18011efc83&m=WUxwcT&r=RaxVT2d).
TASTERMONIAL
